kong/tracing/propagation.lua

```diff
@@ -623,6 +701,13 @@ local function set(conf_header_type, found_header_type, proxy_span, conf_default
     )
   end
 
+  if conf_header_type == "gcp" or found_header_type == "gcp" then
+    set_header("x-cloud-trace-context", fmt("%s/%s;o=%s", 
+      to_gcp_trace_id(to_hex(proxy_span.trace_id)),
+      to_hex(proxy_span.span_id),
```

Comment:
similarly to the comment above, span_id should be set as `from_binary(proxy_span.span_id):to_dec()` here. This might suggest a missing test scenario.
```suggestion
      from_binary(proxy_span.span_id):to_dec(),
```